_dev/src/components/commons/banner-ads.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <b-card
+    class="border-0"
+    :class="{ 'shadow-sm mb-3' : size !== 'xs' }"
+  >
+    <button
+      v-if="isBanner"
+      type="button"
+      class="close"
+      aria-label="Close"
+      @click="closeBanner"
+    >
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <div class="d-flex align-items-center">
```

Comment:
Might not look well on mobile.  
You could stack all element on mobile and adjust margin accordingly.

```suggestion
    <div class="d-flex flex-column flex-md-row align-items-center">
```